database.rules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    ".read": "auth != null",
+    ".write": "auth != null"
+  }
```

Comment:
💯 Good work putting these rules here. Most students don't bother to put this in here. The usually end up putting the rules right in the DB service. 👏